rasa/shared/core/domain.py

```diff
@@ -1810,7 +1804,10 @@ def slot_mapping_for_form(self, form_name: Text) -> Dict[Text, Any]:
         Returns:
             The slot mapping or an empty dictionary in case no mapping was found.
         """
-        return self.forms.get(form_name, {})
+        if self.forms.get(form_name, {}):
+            return self.forms.get(form_name, {})[REQUIRED_SLOTS_KEY]
+        else:
+            return {}
```

Comment:
why do we need this?

Reply:
@Imod7

Reply:
The same as above. If I do not have the check, the same test `pytest tests/core/actions/test_forms.py::test_extract_requested_slot_default` fails because `domain.forms` is an empty dict `{}`.

Reply:
let's fix the test instead as described in the other comment 👍🏻

Reply:
I have a question on that and it is mentioned in the other comment.

Reply:
Adjusted the test by giving a domain with valid slot mappings for the form.